### DISCUSSION TITLE
Como identificar o seu nível profissional?

### DISCUSSION THREAD

**Discussion by viniciusdocanto** [frontendbr/forum#92] [Discussão]
Sempre fico meio em dúvida em relação a minha definição profissional, se sou Frontend Pleno ou Sênior. Tenho já alguns anos de front, mais acho que não tenho conhecimento para uma mudança de nível.
Para vocês qual é a melhor forma de se identificar?

**Comment by felipefialho** [comment]
Cara, acho que a diferença entre um pleno e um sênior está além de fatores técnicos.

Acho que tem mais haver com como o profissional se comporta diante de problemas e na liderança de um time.

A tendencia é que um Sênior também tenha maior controle e precisão sobre o tempo que ele vai gastar nas tarefas, diminuindo assim os custos da empresa.

Outro fator importante,é que com o passar do tempo, os profissionais tendem a ter uma visão cada vez mais macro do produto, agregando valor muito além do código.

Espero que faça sentido ;)

**Comment by rubenmarcus** [comment]
Muita gente considera o fator tempo como classificação ou salario.

Ex:

**Estágiario:** até 1500 r$ - até 1 ano
**Junior:**  de 1500 r$ a 2500 r$ até 2 anos
**Pleno:**  de 3000r$ a 5000 r$ - de 2 a 5 anos
**Senior:**  de 6 a 1XXXX r$  de 6 a XXX anos

Mas o que tabela mesmo são duas coisas: conhecimento e clientes... você pode ter conhecimento bem básico,mas se sabe atrair clientes e vender seu trabalho pode ganhar muito mais que senior. ou o contrario tambem ter muito conhecimento e experiencia e não saber se vender ( o que mais acontece )

Eu mexo com sites desde os 12 anos (2001), mas profissionalmente mesmo comecei em 2011. E tenho ai 5 anos de profissão , mas tiro o salário de um pleno iniciante... o máximo que cheguei foi 4200 PJ. Eu fiquei muitos anos no mundo de startup e agências e perdi um tempão em questão de conhecimento tecnico.

Hoje eu to tirando um valor ainda menor como CLT.. que seria de um front pleno em inicio (3500), o ideal pra mim era tirar no minimo uns 4500 - 5000 , que pelo tempo de experiencia que eu tenho eu julgo estar entre um front pleno médio e um front pleno buscando ser senior já.

vai da sua experiencia com projetos e conhecimentos tecnicos eu vou dividir uma classificação que eu tambem acredito:

**Estagiario:** Ter noções básicas de código e edição de imagens e layout
**Junior:** Básico de JQuery  e JavaScript (interações no layout etc), HTML/CSS ,UI/UX ,GIT Básico
**Pleno:** Frameworks JS,JS puro,Experiencia com Arquitetura de Projeto,Arquitetura de CSS,Foco em Performance, Back-end Junior em alguma linguagem, Modelagem de dados, Pleno em Webdesign e UI/UX (ter um certo dominio sem necessitar da ajuda de um senior
**Senior:** ter dominio de tudo como o @LFeh falou

Uma dica muito importante: Melhor que o salário, ou projeto que você trabalha, é observar como a empresa te trata. Eu trabalhei em empresas por muito tempo em que eu era só um recurso. Não era um funcionário da empresa. Estava ali pra cumprir projetos, e questões como : plano de carreira, valorização profissional, reconhecimento e etc eram apenas discurso. Isso você vai encontrar em muitas empresas. É muito importante estar numa empresa em que você tenha liberdade pra criar e aprender, que tenha uma equipe competente pra trocar informações com você e ambos ensinar, e que também você sinta confiança tanto no lado profissional quanto pessoal.

A Grande maioria das vagas que você vai encontrar são de startups pequenas e de empresas como agências e etc que tem uma  rotatividade muito grande e não oferecem um ganho profissional e pessoal real. Você pode tirar 5000 num mês, mas o stress, as discussões, as noites mal dormidas, a desvalorização pessoal e profissional não compensam.

Mire-se tambem no que você quer se ver realizando em 2 anos, 3, 5, 10... Pegue e leia livros,sites e tutoriais,guias,blogs,vá em eventos, compare seus  conhecimentos profissionais, com o de outros, seu portfolio com o de outros, seu fluxo de trabalho com o de outros, se grave programando, pergunte-se se dá  pra programar melhor em menos tempo, pergunte-se porque está programando.

Acho que assim , você chega numa situação real profissional.

Mas basicamente o feeling é:

**Estagiário:** Ouviu falar
**Junior:** Tem noções e certa independencia
**Pleno:** Tem independencia
**Senior:** Tem Dominio

Mas todos eles consultam o google, stackoverflow e sites nacionais e gringos uahauhauha , vejo isso por varios chefes que tive

**Comment by bernardodiasc** [comment]
> Estagiário: Ouviu falar
> Junior: Tem noções e certa independencia
> Pleno: Tem independencia
> Senior: Tem Dominio

Isso faz sentido... Na minha experiência profissional nunca preocupei com títulos, chegava na entrevista e pedia X de salário que era o que eu queria, falava tudo que manjava lá e boa. Sei lá, fiz poucas entrevistas pq sempre preferi ser freelancer. Mas geralmente nas poucas vezes deu certo, a frase mágica que fazia os olhinhos brilharem é "Não sei mas posso aprender" - e aprendia. :stuck_out_tongue_winking_eye:

**Comment by renanvaz** [comment]
Galera, essa citação aqui é só uma sinopse e não vale para qualquer empresa:

> Estagiário: Ouviu falar
> Junior: Tem noções e certa independencia
> Pleno: Tem independencia
> Senior: Tem Dominio

O que vale ter em mente é que os parâmetros e salários variam de acordo com a estrutura e necessidade da empresa. Não existe uma forma padrão para determinar o seu nível ou o quanto vc deveria estar recebendo. Um exemplo é um cara Sênior em uma produtora digital, pode não ter o conhecimento necessário para ser Sênior no Google, mas pode ser que ele entre no Google ganhando mais do que ele ganhava. Isso depende de quanto do conhecimento que vc possui para o que vc realmente vai precisar fazer. A experiencia conta e o perfil pessoal mais ainda, o seu nível não é apenas o conhecimento técnico que vc possui e sim o conjunto dos fatores que possam dizer que vc é um bom profissional.

**Comment by bernardodiasc** [comment]
Boa sorte mano @rubenmarcus, o que conta é tua postura. Levei um tempo pra aprender isso, no inicio abaixava a cabeça e aceitava o que tinha pro dia. Quando se aprende que o valor é você que se dá, não os outros, a coisa muda de figura. Acho que as vezes quando pinta esses dilemas, o importante é sair da zona de conforto e se desafiar, e desafiar as circunstancias. (e estudar!!!)

Fiz isso em uma ocasião (mais de 7 anos atrás) e mudou minha vida: numa entrevista de emprego que sabia que estava cotada pra 1500 e eu estava disposto a aceitar pois estava na pindaíba, na hora que me perguntaram minha pretensão salarial, ao ver que o meu papo tinha sido bom, lancei 3000 limpo na carteira (taxas a mais). Não peguei a vaga pois pedi muito mais do que estavam dispostos à pagar, evidentemente e esperado, mas saí de lá com o valor que dava pra mim mesmo muito maior, e nunca mais abaixou. Foi foda, fui feliz! A mesma agencia me contratou alguns meses depois pra trabalhar remotamente por 1700 o que foi mil vezes melhor do que viajar pra outra cidade todo dia pra trabalhar período integral por 3k.

Depois de um tempo fiz o seminário do Empretec e lancei uma startup com 2 estagiários, só pra estar na pele de quem paga, pra aprender na prática o outro lado. O aprendizado prático não tá em nenhum livro nem faculdade. Foi legal, mas ser chefe não é meu perfil, sou freelancer no sangue kkkkk

**Comment by viniciusdocanto** [comment]
Fiquei tentando ter um parâmetro por vagas de empregos, sei-lá, vejo umas vagas e me sinto meio medíocre, trabalho com web desde 2007, demorei muito pra investir em mim, mas acho agora que estou nessa de pleno médio pra sênior.
Atualmente me encaixei em uma agência (1 ano e 6 meses nela) que eu tenho espaço para dar ideias e propor soluções e não ser só uma maquina de escrever códigos, o salario é de front pleno em inicio, mas é uma agência saindo do pequeno indo pro médio porte, minhas perspectivas são boas.

**Comment by felipefialho** [comment]
O @renanvaz tocou em um ponto super importante. O seu nível profissional varia muito de acordo com o que as empresas estão buscando.

Você pode ser o sênior pica em uma, mas ser um pleno em outra. Depende muito do seu perfil e como ele se enquadra nos valores da empresa.

**Comment by yanmagale** [comment]
Concordo muito com a opinião de vocês.

Uma coisa que eu acho que ajudaria, era se você conseguisse mapear os "gatilhos" das mudanças de nível, para sentir a sua evolução. Algo que te desse um despertar e você de fato conseguisse perceber: "Opa, estou evoluindo, agora eu não sou mais um júnior". É uma coisa que eu não consigo enxergar. Vocês também veem dessa forma? Sentem essa necessidade?

**Comment by felquis** [comment]
A parte do assunto de "nível profissional", um termo que usam no trabalho é "quando o desenvolvedor está descasado com a realidade", ex: uma pessoa se acha muito boa, mas não é tão boa assim.

Ex: Eu sou o melhor programador. Legal, você escreve testes? Não!

Desculpa, mas estão você não é that good programmer.. Pelo menos é isso que eu tenho visto em empresas grandes, e entre amigos que trabalham em grandes projetos.

Acho que é mais fácil responder **O que não é um nível avançado**, do que responder _O que é nível avançado_.
